shm/GeneralPrint.cpp

```diff
@@ -0,0 +1,79 @@
+#include "GeneralPrint.hpp"
+
+void printResponse(Response response) {
+    std::array text{
+        std::string("done"),
+        std::string("lack_of_money"),
+        std::string("lack_of_cargo"),
+        std::string("lack_of_space")};
+    std::cout << "Response: " << text[static_cast<size_t>(response)] << "\n";
+}
+
+void printPlayerStatus(const Player &player) {
+    std::cout << "Player money: " << player.getMoney() << '\n';
+}
+
+void printPlayerGoal(size_t goal){
+    std::cout << "Goal money: " << goal << '\n';
+}
+
+void printMenu(std::map<std::string, std::unique_ptr<Command>> &commands) {
+    std::cout << "\n\nMENU -------\n";
+    for (const auto &[key, command] : commands) {
+        std::cout << key << " - " << command->getName() << '\n';
+    }
+}
+
+void printStoreCargoList(const Store &store) {
+    printf("\033[31m");
+    std::cout << "STORE LIST -------\n";
+    for (size_t i = 0; i < store.getCargoVec().size(); ++i) {
+        std::cout << i << "\t"                                                 //
+                  << "Name: " << store.getCargoVec()[i]->getName() << "\t\t"   //
+                  << "Amount: " << store.getCargoVec()[i]->getAmount() << "\t" //
+                  << "Price: " << store.getCargoVec()[i]->getPrice() << "\n";  //
+    }
+    printf("\033[0m\n");
+}
+
+void printShipCargoList(const Player &player) {
+    printf("\033[33;44m");
```

Comment:
Please make it constexpr with the proper name like. Special border (it is hard to guess what is "\033[0m\n" ;D